src/privileged/FirefoxMonitor.css

```diff
@@ -0,0 +1,85 @@
+#fxmonitor-notification .popup-notification-body {
+  font-size: 110%;
```

Comment:
Why are you resizing the font size?

Reply:
The spec actually says 13px :-/

I think I'll get rid of this line and ask Eric for comment though.